lib/tasks/edu.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+namespace :edu do
+  desc 'Given a confirmation number, print a spool file entry'
+  task :print, [:id] => [:environment] do |_t, args|
+    raise 'need to give an id. edu:print[{id}]' if args[:id].blank?
+    id = args[:id].gsub(/\D/, '').to_i
```

Comment:
The "confirmation numbers" are output as like `vets_gov_education_benefits_claim_1` so I wanted to make it easy to just copy-paste those in, without needing to grab just the number at the end.